lightningd/peer_control.c

```diff
@@ -1856,3 +1851,13 @@ static const struct json_command fund_channel_command = {
 	"Returns once channel established"
 };
 AUTODATA(json_command, &fund_channel_command);
+
+const char *peer_state_name(enum peer_state state)
```

Comment:
Yes, I think I cut and paste this.  Prefer it fully-generic, it's safer if we delete an element (and shouldn't be a bottleneck).